pom.xml

```diff
@@ -156,6 +156,7 @@ under the License.
 		<hivemetastore.hadoop.version>2.7.5</hivemetastore.hadoop.version>
 		<japicmp.referenceVersion>1.10.0</japicmp.referenceVersion>
 		<japicmp.outputDir>tools/japicmp-output</japicmp.outputDir>
+		<flink.format.parquet.version>1.10.0</flink.format.parquet.version>
```

Comment:
I'd prefer this to be defined in `flink-formats`; to keep things as localized as possible.